src/wp-includes/class-wp-block-patterns-registry.php

```diff
@@ -213,6 +222,25 @@ public function get_all_registered( $outside_init_only = false ) {
 		);
 		$hooked_blocks = get_hooked_blocks();
 		foreach ( $patterns as $index => $pattern ) {
+			if ( ! isset( $pattern['name'] ) ) {
```

Comment:
I don't feel this is not a neat code - I would appreciate suggestions. 

After the current changes to `_register_theme_block_patterns`, `$patterns` had many copies of patterns with blank `content` and no other properties. 
 
I feel the issue was with the removal of `continue;`  however, I was not able to make out how it affects `names` and where the origin of blank content is. 

This `if` is more of a temp fix to get the old test cases running.

Reply:
This sounds like a strange bug to me. Previously, if the content was loaded from the file and the result was no content, the pattern would not be registered at all. By lazily reading that file data, we'll need to remove these when they are encountered during getters. Can you try to share how to reproduce the case you're running into so we can see if additional changes to the `register()` method are needed to validate a pattern before it is added to the registry? That way the first block would become unnecessary.

Reply:
to reproduce error->
comment this block and run 
```
docker-compose run php ./vendor/bin/phpunit --filter test_register_theme_block_patterns_on_init
```

The error gets removed if we revert  back code from 
`[src/wp-includes/block-patterns.php](https://github.com/WordPress/wordpress-develop/pull/5941/files#diff-b4514aa0e877e5200774c5cd82ec0870446f0a93ec7699eaa43c1a33c0d4f74a)`